python/orca/example/learn/mmcv/faster_rcnn/README.md

```diff
@@ -0,0 +1,170 @@
+# MMCV Faster-RCNN Training Example
+
+## Prepare the dataset
+
+This training example use the kitti_tiny dataset, you can download it from [here](https://download.openmmlab.com/mmdetection/data/kitti_tiny.zip) and unzip the dataset.
+
+## Prepare the environment
+
+We recommend you to use Anaconda to prepare the environment:
+
+```
+conda create -n bigdl-orca pyton=3.7  # "bigdl-orca" is conda environment name, you can use any name you like.
+conda activate bigdl-orca
+
+pip install torch torchvision tensorboard
+pip install -U openmim
+mim install mmcv-full
+pip install mmdet
+
+pip install --pre --upgrade bigdl-orca
+pip install ray
+```
+
+## Download MMDet
+
+In this example, we need to use the config file from the [mmdet repo](https://github.com/open-mmlab/mmdetection).
+
+```
+git clone https://github.com/open-mmlab/mmdetection
+```
+
+The config file we use in this example is `mmdetection/configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_1x_coco.py`
+
+If you want to use the pre-trained checkpoints for further training, also download the checkpoint file
+
+```
+cd mmdetection
+mkdir checkpoints
+wget https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_3x_coco/faster_rcnn_r50_caffe_fpn_mstrain_3x_coco_20210526_095054-1f77628b.pth
+```
+
+## Running commands
+
+- spark local mode
```

Comment:
keep this format the same as other examples? e.g. add arguments explanation; put the results at the very end